boards/rapidsilicon/virgo_proto/virgo_proto.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 Rapid Silicon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /dts-v1/;
+
+ #include <rapidsilicon/rapidsi_virgo.dtsi>
+ /
+ {
+	 model = "Rapid Silicon Virgo Proto";
+	 compatible = "rapidsi,virgo_proto";
+	 timebase-frequency = <26666667>; // 26.6MHz for fpga only
+ 
+	 chosen {
+		 zephyr,console = &uart0;
+		 zephyr,shell-uart = &uart0;
+		 zephyr,sram = &sram;
+		 zephyr,flash = &xip_mem;
```

Comment:
```suggestion
		 zephyr,flash = &flash;
```
I think this should be flash and not xip_mem because when xip mode is on as per the soc/rapidsilicon/virgo/linker.ld we want to execure from flash (which we dont allow currently). Actually xip_mem is a misname, but I dont know what the name should be so just keep it as it is

Reply:
Done